docs/dense-retrieval.md

```diff
@@ -166,6 +166,31 @@ To evaluate with on-the-fly query encoding with our pretrained encoder model
 on [Hugging Face](https://huggingface.co/castorini/tct_colbert-msmarco/tree/main) add
 `--encoder castorini/tct_colbert-msmarco`. The encoding will run on CPU by default. To enable GPU, add `--device cuda:0`.
 
+## TREC2019-DL Document Ranking (Zero Shot)
+TREC-2019 DL document ranking task, dense retrieval with TCT-ColBERT trained on MS MARCO passages, brute force index.
```

Comment:
actually, TREC 2019 is for our internal sanity check - externally, I'd rather present replication on MS MARCO doc.

Reply:
sure, I'll change to msmarco